docs/wallet-integration-guide/examples/scripts/exchange-integration/01-one-step-deposit.ts

```diff
@@ -3,43 +3,34 @@ import { pino } from 'pino'
 import { v4 } from 'uuid'
 import { setupExchange } from './setup-exchange.js'
 import { setupCustomer } from './setup-customer.js'
+import { tapParty } from './tap-party.js'
+import { validateTransferIn } from './validate-transfers.js'
 
-const logger = pino({ name: '08-one-step-deposit', level: 'info' })
+const logger = pino({ name: '01-one-step-deposit', level: 'info' })
 
-const { exchangeParty, treasuryParty, exchangeSdk } = await setupExchange()
+const { exchangeParty, treasuryParty, exchangeSdk } = await setupExchange(true)
```

Comment:
consider using a named argument to make it clear that this is w/o transfer preapprovals